media/css/mozorg/contribute/friends.less

```diff
@@ -0,0 +1,153 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+.firefox-friends {
+    #wrapper {
+        padding-bottom: 0;
+        width: 100%;
+    }
+    #masthead {
+        background: #373737 url(/media/img/contribute/friends/header-img-wide.jpg) 50% 0 no-repeat;
+        padding: 0 180px;
```

Comment:
Would it be easier to set a fixed width here on `#masthead` instead of adjust the padding at different screen sizes? This would probably make it more consistent with other pages.